src/main/java/mate/academy/spring/validator/RepeatPassword.java

```diff
@@ -0,0 +1,19 @@
+package mate.academy.spring.validator;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import javax.validation.Constraint;
+import javax.validation.Payload;
+
+@Documented
```

Comment:
Not sure if this annotation is necessary. Let the mentors decide.

Reply:
I've seen examples that this annotation is used in password validators, but I really think it's better to ask a mentor's opinion here